graphics/timg/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github  1.0
+
+github.setup        hzeller timg 1.6.0 v
+github.tarball_from archive
+revision            0
+categories          graphics
+
+maintainers         {outoforder.cc:urkle @urkle} openmaintainer
+
+description         Terminal Image and Video Viewer
+long_description    A user-friendly terminal image viewer that uses graphic \
+                    capabilities of terminals (Sixel, Kitty or iterm2), or 24-Bit \
+                    color capabilities and unicode character blocks if these are not \
+                    available.
+
+license             GPL-2
+homepage            https://timg.sh/
+
+
+checksums           rmd160  fc1b39f4395f7a4dc4da77f12e558289e05f3425 \
+                    sha256  9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af \
+                    size    1166246
+
+installs_libs       no
+
+depends_build-append \
+                    path:bin/pkg-config:pkgconfig
+
+depends_lib-append  port:GraphicsMagick \
+                    port:webp \
+                    path:include/turbojpeg.h:libjpeg-turbo \
+                    port:libexif \
+                    path:lib/pkgconfig/librsvg-2.0.pc:librsvg \
+                    path:lib/pkgconfig/cairo.pc:cairo \
+                    port:ffmpeg \
+                    port:openslide \
+                    path:lib/pkgconfig/poppler.pc:poppler \
+                    port:pandoc
```

Comment:
`pandoc` is quite a "heavy" dependency in that it pulls in a lot of other stuff. There is no need to regenerate the man page as it is already part of the distribution. So please remove this dependency.

Reply:
I naively followed the instructions on the timg side on needed dependencies :) And yeah I do agree that is a very heavy dependency.